### ISSUE TITLE
NhoodGroups into colData

### ISSUE THREAD

**Issue by AnaBVA** [MarioniLab/miloR#200]
Hi, 

Thank you for creating and maintaining `miloR` package.  
I probably missed this, but I couldn't find it. Is there any way that I can put the "NhoodGroup" groups (calculated with `groupNhoods` ) into `colData(milo)`. I'd like to use the cluster groups for other visualizations (violin plots, expression umaps) and analysis.  

Thank you.

**Comment by emdann** [comment]
Hi @AnaBVA, because the group assignment happens at the level of neighbourhoods and cells can belong to multiple neighbourhoods, there is no straightforward way to add the neighbourhood group information in a single column of `colData(milo)`. Alternatively, depending on your application, you could encode information about which cells belong to which nhood group in a binary matrix, with nhoodGroups as columns and cells as rows.

**Comment by AnaBVA** [comment]
I see! I understood clustering was at neighbourhoods level but didn't catch that a cell could belong to multiple clusters. Thanks, @emdann for pointing it out. 

I wanted to use these groups for clustering but maybe not the best approach. If I have my own groups/clusters, can I use them for analyzing DA with `miloR`? Should I follow [5.0.1 Custom grouping](https://rawcdn.githack.com/MarioniLab/miloR/7c7f906b94a73e62e36e095ddb3e3567b414144e/vignettes/milo_gastrulation.html#501_Custom_grouping) steps?

**Comment by emdann** [comment]
Yes, you can "annotate" neighbourhoods based on the group/cluster identity of the cells that belong to the neighbourhood (usually > 90% of cells in a neighbourhood have the same cell identity). This is implemented in the function `annotateNhoods` (see example `here`).

**Comment by AnaBVA** [comment]
Thanks!
